letsencrypt/cli.py

```diff
@@ -695,6 +725,168 @@ def install(config, plugins):
     le_client.enhance_config(domains, config)
 
 
+def _restore_required_config_elements(full_path, config, renewalparams):
+    # string-valued items to add if they're present
+    for config_item in STR_CONFIG_ITEMS:
+        if config_item in renewalparams:
+            value = renewalparams[config_item]
+            # Unfortunately, we've lost type information from ConfigObj,
+            # so we don't know if the original was NoneType or str!
+            if value == "None":
+                value = None
+            setattr(config.namespace, config_item, value)
+    # int-valued items to add if they're present
+    for config_item in INT_CONFIG_ITEMS:
+        if config_item in renewalparams:
+            try:
+                value = int(renewalparams[config_item])
+                setattr(config.namespace, config_item, value)
+            except ValueError:
+                logger.warning("Renewal configuration file %s specifies "
+                               "a non-numeric value for %s. Skipping.",
+                               full_path, config_item)
+                raise
+
+def _restore_plugin_configs(config, renewalparams):
```

Comment:
Suggestion: Couldn't we add `_parser` to `config` to avoid making it a global? Additionally, could we only store the parser's actions because that's all we care about?